headlines.js

```diff
@@ -0,0 +1,116 @@
+class Headlines {
+  constructor() {
+    this.response = null;
+    this.mainContainerEl = document.querySelector("#main-container");
```

Comment:
really good use of constructor variables, initialising, this.elements = null was also quite smart as it solves a-lot of problems later on and I would have also put newsObjects into an array